code/datums/ruins/lavaland.dm

```diff
@@ -237,3 +237,11 @@
 	description = "Mystery to be solved."
 	suffix = "lavaland_surface_puzzle.dmm"
 	cost = 5
+
+/datum/map_template/ruin/lavaland/elephant_graveyard
+	name = "Elephant Graveyard"
+	id = "Graveyard"
+	description = "An abandonded graveyard, calling to those unable to continue."
```

Comment:
abandonded > abandoned

Reply:
Oh you think I just spelled one thing wrong? Oh boy get reading then, I'm sure I fucked something else somewhere. (Thanks)

Reply:
a**bone**doned